src/test/java/app/coronawarn/verification/VerificationApplicationTest.java

```diff
@@ -24,16 +24,7 @@
 import app.coronawarn.verification.config.VerificationApplicationConfig;
 import app.coronawarn.verification.domain.VerificationAppSession;
 import app.coronawarn.verification.domain.VerificationTan;
-import app.coronawarn.verification.model.AppSessionSourceOfTrust;
-import app.coronawarn.verification.model.AuthorizationRole;
-import app.coronawarn.verification.model.HashedGuid;
-import app.coronawarn.verification.model.RegistrationToken;
-import app.coronawarn.verification.model.RegistrationTokenKeyType;
-import app.coronawarn.verification.model.RegistrationTokenRequest;
-import app.coronawarn.verification.model.Tan;
-import app.coronawarn.verification.model.TanSourceOfTrust;
-import app.coronawarn.verification.model.TanType;
-import app.coronawarn.verification.model.TestResult;
+import app.coronawarn.verification.model.*;
```

Comment:
I recommend the import of classes and not an entire package

Reply:
I've rearranged the imports for this file